src/js/view/controls/controls.js

```diff
@@ -161,21 +161,23 @@ export default class Controls {
             this.div.insertBefore(settingsMenu.element(), controlbar.element());
         }
 
-        // Unmute Autoplay Button. Ignore iOS9. Muted autoplay is supported in iOS 10+
-        if (model.get('autostartMuted')) {
-            const unmuteCallback = () => this.unmuteAutoplay(api, model);
-            this.mute = button('jw-autostart-mute jw-off', unmuteCallback, model.get('localization').unmute,
-                [cloneIcon('volume-0')]);
-            this.mute.show();
-            this.div.appendChild(this.mute.element());
-            // Set mute state in the controlbar
-            controlbar.renderVolume(true, model.get('volume'));
-            // Hide the controlbar until the autostart flag is removed
-            utils.addClass(this.playerContainer, 'jw-flag-autostart');
-
-            model.on('change:autostartFailed change:autostartMuted change:mute', unmuteCallback, this);
-            this.unmuteCallback = unmuteCallback;
-        }
+        // Unmute Autoplay Button.
+        model.once('change:autostartMuted', (_model, muted) => {
```

Comment:
If "autostartMuted" is true when controls are added, this should be invoked immediately, but it will not be. YOu can use `model.change('autostartMuted'` to help with this, but then you have to manage removal of the "change:" listeners.

Controls can be added and removed from the player with `jwplayer().setControls(bool)`. Each time controls are readded, this code is rerun with the current model state - so you could have missed the one "change:" event.